examples/lit/js/components/TodoItem.js

```diff
@@ -0,0 +1,97 @@
+import { connect } from 'pwa-helpers'
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+
+import { store, setEdit, setTodo, toggleTodos, leaveEdit, deleteTodo } from '../store.js'
+import { appCSSRules, baseCSSRules } from '../constant.js'
+
+const ESCAPE_KEY = 27;
+
+export class TodoItem extends connect(store)(LitElement) {
+    static properties = {
+        todo: { type: Object }
+    }
+
+    static styles = [
+        css`
+        :host(:focus) { box-shadow: none!important; }
+        .todo-list li:last-child {
+            border-bottom: 1px solid #ededed!important;
+        }
+        `,
+        ...baseCSSRules.map((r) => unsafeCSS(r)),
+        ...appCSSRules.map((r) => unsafeCSS(r))
+    ]
+
+    get isEditing () {
+        const state = store.getState()
+        return this.todo.id === state.editingTodo
+    }
```

Comment:
I think it would be better to `connect` the TodoList component, and pass `isEditing` as a boolean property. 
This way we wouldn't rerender all the items (see `requestUpdate` below) when the state of just one item changes, or even the state of any other thing that's not related to these items.

Note that I know very little about Lit so I may be off here.